test/lang/test_skip_tokenizer_srt.py

```diff
@@ -0,0 +1,77 @@
+import json
```

Comment:
For unittest, please do not use `test/lang`.
Can you add a unit test similar to this one https://github.com/sgl-project/sglang/blob/main/test/srt/test_srt_endpoint.py?

Reply:
> For unittest, please do not use `test/lang`. Can you add a unit test similar to this one https://github.com/sgl-project/sglang/blob/main/test/srt/test_srt_endpoint.py?

done. moved to test/srt.